<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tlinsWPilotRaDcIncrement</class>
 <widget class="QWidget" name="tlinsWPilotRaDcIncrement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>Axis:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioRA">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string> Hour angle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioDC">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string> Declination</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupRA">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>Direction Hour Angle:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioRaLeft">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string> Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioRaRight">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string> Right</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupDC">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>Direction Declination:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioDcUp">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string> Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioDcDown">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string> Down</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioRA</sender>
   <signal>clicked()</signal>
   <receiver>tlinsWPilotRaDcIncrement</receiver>
   <slot>raAxisSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioDC</sender>
   <signal>clicked()</signal>
   <receiver>tlinsWPilotRaDcIncrement</receiver>
   <slot>dcAxisSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>raAxisSelected()</slot>
  <slot>dcAxisSelected()</slot>
 </slots>
</ui>
